docs/pages/reference/mesheryctl/mesheryctl-app-list.md

```diff
@@ -48,7 +48,7 @@ mesheryctl app list
 
 <pre class='codeblock-pre'>
 <div class='codeblock'>
-      --config string   path to config file (default "/home/runner/.meshery/config.yaml")
+      --config string   path to config file (default "/home/admin-pc/.meshery/config.yaml")
```

Comment:
@alphaX86 Could you please let me know the reason behind the /admin-pc/ ? 👀

Reply:
@adithyaakrishna Ah, seems it's referenced to my system path. Don't worry once the CI action is made to run after merge, it'll automatically revert to default path name